Fix identification of OS name when OS version contains non-digit non-period characters.
https://github.com/nigels-com/glew/pull/389
--- Makefile.orig	2020-03-15 06:53:59.000000000 -0500
+++ Makefile	2023-07-24 20:45:57.000000000 -0500
@@ -31,7 +31,7 @@
 include config/version
 
 SHELL = /bin/sh
-SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/[0-9\.]//g;')
+SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/\.//g' -e 's/[0-9]\{1,\}.*//')
 SYSTEM.SUPPORTED = $(shell test -f config/Makefile.$(SYSTEM) && echo 1)
 
 ifeq ($(SYSTEM.SUPPORTED), 1)
